src/modules/counter/CounterView.js

```diff
@@ -27,7 +27,10 @@ const CounterView = React.createClass({
     this.props.dispatch(CounterState.random());
   },
   bored() {
-    this.props.dispatch(NavigationState.pushRoute({key: 'Color'}));
+    this.props.dispatch(NavigationState.pushRoute({
+      key: 'Color',
+      title: 'Color Screen'
+    }));
   },
```

Comment:
fixing the `NavigationHeader` title display method means that now each route pushed must have a `title`